test/remote/gateways/remote_nuvei_test.rb

```diff
@@ -416,13 +416,13 @@ def test_successful_purchase_with_google_pay
   end
 
   def test_purchase_account_funding_transaction
-    response = @gateway.purchase(@amount, @credit_card, @options.merge(is_aft: true))
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(is_aft: true, aft_recipient_first_name: 'John', aft_recipient_last_name: 'Doe'))
```

Comment:
❓ Just an small question, what's the difference in passing this and not doing it? Looks like before the change this was working.

💭  if it works any way, maybe a test that demonstrate it could works.